frontend/src/metabase/xray/components/StatGroup.jsx

```diff
@@ -12,14 +12,14 @@ const StatGroup = ({ heading, xray, stats, showDescriptions }) =>
             <div className="bordered rounded shadowed bg-white">
                 <ol className="Grid Grid--1of4">
                     { stats.map(stat =>
-                        !!xray[stat] && xray[stat].value && (
+                        (!!xray[stat] && xray[stat].value) ? (
                             <li className="Grid-cell p1 px2 md-p2 md-px3 lg-p3 lg-px4 border-right border-bottom" key={stat}>
                                 <SimpleStat
                                     stat={xray[stat]}
                                     showDescription={showDescriptions}
                                 />
                             </li>
-                        )
+                        ) : null
```

Comment:
Personally I preferred this how it was, it was more readable than the ternary operator if only because there's 7 lines between when it starts and when it ends. Ternaries get hard to read when they span that many lines

Reply:
The idea here is that in some instances  `xray[stat]` had a value but `xray[stat].value` was missing, causing the value of `xray[stat]` to leak to html tree because of how `&&` operator functions.